zerver/middleware.py

```diff
@@ -210,12 +223,14 @@ class LogRequests(object):
     # for some views, process_view isn't run, so we call the start
     # method here too
     def process_request(self, request):
+        # type: (WSGIRequest) -> None
         request._log_data = dict()
         record_request_start_data(request._log_data)
         if connection.connection is not None:
             connection.connection.queries = []
 
     def process_view(self, request, view_func, args, kwargs):
+        # type: (WSGIRequest, Callable[..., None], List[str], Dict[str, Any]) -> None
```

Comment:
I changed it to `Callable[[HttpRequest, ...], HttpResponse]` and I got
`zerver/middleware.py:231: error: Unexpected '...'`